example/examplejob.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: paddlepaddle.org/v1
+kind: TrainingJob
+metadata:
+  name: example
+spec:
+  #image: "paddlepaddle/paddlecloud-job"
+  image: "typhoon1986/paddlecloud-job:wuyitest"
```

Comment:
Maybe we can keep the official Docker image.

Reply:
Done